hotsos/core/host_helpers/cli.py

```diff
@@ -491,6 +491,10 @@ def command_catalog(self):
                  # requires sosreport 4.3 or above
                  FileCmd('sos_commands/ceph_osd/'
                          'ceph_daemon_osd.{osd_id}_config_show')],
+            'ceph_health_detail':
```

Comment:
Is this info not available in `ceph_health_detail_json_decoded`?

We mostly use json formatted output so that parsing is robust. So if it's available in `ceph_health_detail_json_decoded`, we don't need to use the plain-text version.

Reply:
I'm trying to understand how the error will look in ceph_health_detail_json_decoded, but there's no available sos, so to be safe I thought I'd use ceph_health_detail and just use the error string that Dan put in the issue description.

Reply:
done using ceph_health_detail_json_decoded